src/lib/chips/chip-list.spec.ts

```diff
@@ -63,6 +63,16 @@ describe('MatChipList', () => {
       it('should add the `mat-chip-list` class', () => {
         expect(chipListNativeElement.classList).toContain('mat-chip-list');
       });
+
+      it('should not have the aria-selected attribute when is not selectable', () => {
+        testComponent.selectable = false;
+        fixture.detectChanges();
+
+        chips.forEach(chip => {
```

Comment:
This one will end up generating a bunch of failure messages if we ever end up breaking the functionality. Something like this might be cleaner:

```ts
const chipsValid = chips.every(chip => !chip.selectable && !chip._elementRef.nativeElement.hasAttribute('aria-selected')):

expect(chipsValid).toBe(true);
```